src/Swarm/TUI/Controller.hs

```diff
@@ -320,6 +320,8 @@ handleMainEvent ev = do
     -- toggle creative mode if in "cheat mode"
     ControlChar 'v'
       | s ^. uiState . uiCheatMode -> gameState . creativeMode %= not
+    -- toggle collapse/expand REPL
+    ControlChar 's' -> uiState . uiShowREPL %= not
```

Comment:
We should add a call to `invalidateCacheEntry WorldCache` here, so it redraws the world instantly instead of waiting for the next time it decides to redraw.  The idea is that the world map is cached, so that we don't need to redraw it every frame if it hasn't changed.  This probably saves a bunch of cycles when nothing is going on, but it is also kind of a pain to deal with figuring out all the different places where the cache needs to be invalidated.